src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -2,7 +2,23 @@
 
 public class WorkWithFile {
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String COMMA = ",";
+    public static final String SPACE = " ";
+    public static final int DEFAULT_NUMBER = 0;
```

Comment:
it would be better to move these constants to classes where they are used and make them `private`